File: net/ShopErp.App/Service/Restful/ShopService.cs
using ShopErp.Domain;
using ShopErp.Domain.RestfulResponse;
using System;
using System.Collections.Generic;
using System.Linq;

namespace ShopErp.App.Service.Restful
{
    public class ShopService : ServiceBase<Shop>
    {
        public DataCollectionResponse<Shop> GetByAll()
        {
            return DoPost<DataCollectionResponse<Shop>>(null);
        }

        public StringResponse GetShopOauthUrl(long shopId)
        {
            Dictionary<string, object> para = new Dictionary<string, object>();

            para["shopId"] = shopId;

            return DoPost<StringResponse>(para);
        }
    }
}

File: android/app/src/main/java/bjc/shoperp/service/restful/ShopService.java
package bjc.shoperp.service.restful;

import java.util.HashMap;
import java.util.List;

import bjc.shoperp.domain.Shop;
import bjc.shoperp.domain.restfulresponse.domainresponse.ShopCollectionResponse;

/**
 * Created by hcq on 2018/2/20.
 */

public class ShopService extends ServiceBase<Shop> {

    public List<Shop> getByAll() throws Exception {
        HashMap<String, Object> para = new HashMap<>();
        return DoPost(ShopCollectionResponse.class, para, null).Datas;
    }
}
